package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "expensify.cash",
-  "version": "1.0.1-484",
+  "version": "1.0.1-485",
```

Comment:
Testing? Should we revert?

Reply:
So annoying how running `npm version` commits on its own by default 😠